aiohttp/hdrs.py

```diff
@@ -11,6 +11,7 @@
 METH_POST = upstr('POST')
 METH_PUT = upstr('PUT')
 METH_TRACE = upstr('TRACE')
+METH_PURGE = upstr('PURGE')
```

Comment:
Looks like unrelated change. PURGE is used for Varnish needs, but what's it purpose for aiohttp?